master/utils/doregister.py

```diff
@@ -0,0 +1,28 @@
+from flask import Flask, flash, redirect, render_template, request, session, abort, url_for
+from db.registrationClass import *
+from db.db import *
+
+def do_register(Form): 
+    form = RegistrationForm(request.form)
+    # use form from register.html and pull data from that form
+    if request.method == "POST" and form.validate():
```

Comment:
if this condition isn't met, it doesn't look like you return True or False, even though you return a boolean when the condition is met. 

what you can do is create 'status = false' or 'status = 'true' variable at the start of the function, depending on whatever should be the default return value. in your code you can then reset status as needed, depending on the condition statements below. the only return statement you'll have is at the very end of the function 'return status'
 
    status = false  
        if condition:   
            do stuff
            if condition:  
                logger.error(msg)  
            else:  
                status = true  
    return status